### PR TITLE
Re-enable NyanCat

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
Some crazy person spend time making the thing work with PHPUnit 7.x :)

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
image

File: phpunit.xml.dist [hunk 1]
```diff
@@ -10,7 +10,8 @@
          stopOnFailure="false"
          stopOnIncomplete="false"
          stopOnSkipped="false"
-         verbose="false">
+         verbose="false"
+         printerClass="NyanCat\PHPUnit\ResultPrinter">
```
Comment (gbirke) [inline phpunit.xml.dist]: This will make the output on Travis unnecessarily verbose (and not nyan-pretty). +2 for the rest of the PR.

**Comment by JeroenDeDauw** [comment]
@tzhelyazkova huh? For me the tests take 1.4 minutes with or without this. Different for you?

File: phpunit.xml.dist (hunk 1 above)
Comment (JeroenDeDauw) [reply phpunit.xml.dist]: I changed it so the CI is not nyan

**Comment by gbirke** [comment]
I have tested both this branch and the master branch twice and did not get different results (and a variation of ±5 seconds during all tests).

As a side note - I prefer the ugly version, since it shows the progress in percent.

**Comment by JeroenDeDauw** [comment]
> shows the progress in percent.

I hear a feature request for the nyan printer :D

**Comment by JeroenDeDauw** [comment]
Maybe @tzhelyazkova accidentally once with xdebug and once without?

**Comment by tzhelyazkova** [comment]
> Maybe @tzhelyazkova accidentally once with xdebug and once without?
Don't know if that's the problem but yeah the phpunit tests are generally slow on my machine, nyan cat or not.

**Comment by JeroenDeDauw** [comment]
I was also surprised by how slow they where for this git repo, though thinking about it, it does make sense: these are basically all edge-to-edge tests.

@tzhelyazkova how long do they take for you to run?

**Comment by JeroenDeDauw** [comment]
image

**Comment by timEulitz** [comment]
tieu@tieu-ThinkPad ~/projects/FundraisingFrontend **(master)** $ make phpunit
**Time: 15.88 seconds, Memory: 50.25MB**

tieu@tieu-ThinkPad ~/projects/FundraisingFrontend **(new-nyan)** $ make phpunit
**Time: 16.45 seconds, Memory: 50.25MB**

Actually kind of crazy that it's taking 1.4 minutes on your machine @JeroenDeDauw, are you sure you did not accidentally SSH into your kitchen toaster and ran the tests from there 😬?

At the end of the day, I don't mind this change but I also don't need it, if it's causing performance issues on some setups I'd rather not implement it cause that would definitely be more important.

**Comment by tzhelyazkova** [comment]
On my machine,
 :older_man:  style:
```tozh@C251:~/Documents/Projects/playground/FundraisingFrontend$ make test
PHPUnit 7.3.5 by Sebastian Bergmann and contributors.

...............................................................  63 / 484 ( 13%)
............................................................... 126 / 484 ( 26%)
............................................................... 189 / 484 ( 39%)
............................................................... 252 / 484 ( 52%)
............................................................... 315 / 484 ( 65%)
............................................................... 378 / 484 ( 78%)
............................................................... 441 / 484 ( 91%)
...........................................                     484 / 484 (100%)

Time: 21.06 seconds, Memory: 50.25MB
```

:rainbow: :cat: 
```
tozh@C251:~/Documents/Projects/playground/FundraisingFrontend$ make test
PHPUnit 7.3.5 by Sebastian Bergmann and contributors.

 100% _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_,------,  
 484  _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_|  /\_/\  
 0    _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-~|_( ^ .^) 
 0    _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_- ""  ""    

Time: 22.32 seconds, Memory: 50.25MB
```

So it's like a second slower. 
I also don't mind it, generally.

**Comment by JeroenDeDauw** [comment]
It might be a second slower but it will feel a lot faster ;)

**Comment by JeroenDeDauw** [comment]
> Actually kind of crazy that it's taking 1.4 minutes on your machine @JeroenDeDauw, are you sure you did not accidentally SSH into your kitchen toaster and ran the tests from there?

I'll check again. Weird they are running slower for me. I would expect my PC to run things faster than your laptop.

**Comment by JeroenDeDauw** [comment]
That is also what I have

image

And the file is definitely used, since if I replace the driver name by derp, the tests fail.

**Pull request merged**